packages/jsii-reflect/test/__snapshots__/jsii-tree.test.ts.snap

```diff
@@ -1712,10 +1712,10 @@ exports[`jsii-tree --all 1`] = `
  │   │   ├── <initializer>() initializer (stable)
  │   │   └─┬ youSeeMe property (stable)
  │   │     └── type: string
- │   ├─┬ class StructPassing
+ │   ├─┬ class StructPassing (external)
```

Comment:
Example of a "fixed bug" (really a feature gap) caused by a non-exhaustive `switch`.